src/app/features/data-import/data-import.service.ts

```diff
@@ -32,7 +34,9 @@ export class DataImportService {
           delete record[propertyName];
         }
       }
-
+      if (record["name"] !== undefined) {
+        record["searchIndices"] = record["name"].split(" ");
```

Comment:
I like the simple approach. 

There is only the problem, that some subclasses of `Entity` override the `searchIndices` function. E.g. the `Child` class includes the `projectNumber`.
To correctly support that you can use the `_id` field of the `record` and the `DynamicEntityService.getEntityConstructor` function to access the entity's constructor and then using something like `Object.assign(new entityContructor(), record).searchIndices`.

Let me know if you need help with that. We can also do this in pair programming!

Reply:
I added a test for it now, as we want to get this bugfix release.